geoapps/inversion/constants.py

```diff
@@ -154,8 +154,6 @@
         "group": "Mesh and Models",
         "main": True,
         "enabled": True,
-        "dependency": "mesh",
```

Comment:
Should we not remove those meshing parameters altogether?

Reply:
yeah... maybe the octree ones, but we need the ones that are used in the pseudo3d ui.json